gallery/sbol_visual/sbol_visual.py

```diff
@@ -1,17 +1,17 @@
-__author__ = 'user'
+__author__ = "user"
 
 import dnaplotlib as dpl
 import dnaplotlib.sbol as dpl_sbol
-import sbol as sbol
+import sbol3 as sbol
 import matplotlib.pyplot as plt
 
 # Import the SBOL design file
 doc = sbol.Document()
-doc.read('gene_cassette.sbol')
+doc.read("gene_cassette.xml")
 
-# In this case, we know ahead of time the URI of the design.  In some cases, you may have to explore the doc's components to find the design you are looking for
-design = doc.components['http://sbolstandard.org/examples/Design']
-print(design.display_id)
+# In this case, we know ahead of time the name of the design. In some cases, you may have to explore the doc's components to find the design you are looking for
+design = doc.find("GeneCassette")
+print(design.name)
```

Comment:
according to the SBOLv3.1 standard it is the name that is used for display purposes
see https://sbolstandard.org/docs/SBOL3.1.0.pdf
page 15